setup.py

```diff
@@ -4,13 +4,25 @@
     long_description = fh.read()
 
 INSTALL_REQUIRES = [
-    'endaq-calc>=1.1.0.post2',
-    'endaq-cloud>=1.1.0',
-    'endaq-ide>=1.1.0',
-    'endaq-plot>=1.1.0',
+    "backports.cached-property; python_version<'3.8'",
+    "ebmlite>=3.2.0",
+    "idelib>=3.2.3",
+    "jinja2",
+    "numpy>=1.19.5",
+    "pandas>=1.3",
+    "plotly>=5.3.1",
+    "python-dotenv>=0.18.0",
+    "requests>=2.25.1",
+    "scipy>=1.7.1",
     ]
 
 TEST_REQUIRES = [
+    "hypothesis",
+    "numpy-quaternion==2020.11.2.17.0.49",
+    "plotly",
```

Comment:
remove unnecessary test requirements (due to removing `endaq-batch`)?
```suggestion
```

Reply:
addressed in a7ad2fe97eecb081fd0361ea02360240d15713fc & 8461b0c63347c7607bc29b54840af15e79d51bb1